src/root.h

```diff
@@ -69,6 +71,9 @@ class Root : public Controller
 private:
     QElapsedTimer m_cleaupTimer;
     HtmlHighlighter *m_htmlHighlighter;
+    bool m_socialMediaButtons = 0;
```

Comment:
you should either initialize to true or false, or since you will initilize in prefork just don't initialize.